packages/theme/include/icon.ts

```diff
@@ -0,0 +1,8 @@
+export const mediumHeight = {
+  height: 22
+}
+
+//icon images
+export const icStats = 'md-stats'
+export const icPeople = 'md-people'
+export const icCash = 'md-cash'
```

Comment:
These constants add no value and should be removed. `icCash` is not shorter than `md-cash` and adds an import.